CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ option(TILEDB_S3 "Enables S3/minio support using aws-cpp-sdk" OFF)
 option(TILEDB_AZURE "Enables Azure Storage support using azure-storage-cpp" OFF)
 option(TILEDB_GCS "Enables GCS Storage support using google-cloud-cpp" OFF)
 option(TILEDB_HDFS "Enables HDFS support using the official Hadoop JNI bindings" OFF)
+option(TILEDB_MEMFS "Enables support of an in-memory filesystem." OFF)
```

Comment:
What is the motivation behind disabling memfs by default?

Reply:
My initial thought too.  So external build requirements so why not turn it on automagically?

Reply:
Discussed offline -- resolving conversation